tests/test_output.py

```diff
@@ -0,0 +1,60 @@
+import sys
+
+import pytest
+
+from check_oldies import output
+
+
+# `xml.etree.ElementTree.tostring()` preserves attribute order only as of
+# Python 3.8. With prior versions, we cannot easily compare XML
+# output, so don't bother.
+HAS_UNSORTED_XML_WRITE = sys.version_info[:2] < (3, 8)
```

Comment:
Python minimal version supported is 3.8. So we can remove all logic link to that

Reply:
Oui, dans une autre pull request où je supprimerai le support de Python 3.7 (entre autres modernisations).